### PR TITLE
ADS release notes v2.8.5

### PR DESCRIPTION
**Pull request by liudmylaru**
### Description

ADS release notes v2.8.5

Jira: https://jira.oci.oraclecorp.com/browse/ODSC-42778

### What was done

- added release notes
- bumped version to 2.8.5
- minor fix in docs about python version prerequisite for ADS CLI

### CODE REVIEW

**Review by lvallad — APPROVED** [review]

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -2,6 +2,16 @@
 Release Notes
 =============
 
+2.8.5
+-----
+Release date: May 17, 2023
+
+* Added support for ``key_content`` attribute in the ``ads.set_auth`` for the API KEY authentication.
```
[the remark this answers is not in the document]
Comment (liudmylaru) [reply docs/source/release_notes.rst]: updated

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
+* Added support for the mounting file systems in the Jobs API.
```
[the remark this answers is not in the document]
Comment (liudmylaru) [reply docs/source/release_notes.rst]: updated

File: docs/source/release_notes.rst [hunk 3]
```diff
@@ -2,6 +2,16 @@
...
+* Fixed bug in ``ModelEvaluator`` when it returned incorrect ROC AUC characteristics.
+* Fixed bug in ``ADSDataset.suggest_recommendations`` API, when it returned a error if target was not specified.
+* Fixed bug in ``ADSDataset.auto_transform`` API, when an incorrect sampling was suggested for imbalanced data
```
[the remark this answers is not in the document]
Comment (liudmylaru) [reply docs/source/release_notes.rst]: updated

File: docs/source/release_notes.rst [hunk 4]
```diff
@@ -2,6 +2,16 @@
...
 =============
 
+2.8.5
+-----
+Release date: May 17, 2023
+
+* Added support for ``key_content`` attribute in the ``ads.set_auth`` for the API KEY authentication.
+* Added support for the mounting file systems in the Jobs API.
+* Fixed bug in ``ModelEvaluator`` when it returned incorrect ROC AUC characteristics.
+* Fixed bug in ``ADSDataset.suggest_recommendations`` API, when it returned a error if target was not specified.
```
[the remark this answers is not in the document]
Comment (liudmylaru) [reply docs/source/release_notes.rst]: updated

**Review by mrDzurb — APPROVED** [review]

**Review by qiuosier — APPROVED** [review]

**Review by qiuosier — APPROVED** [review]

**Pull request merged**